### PR TITLE
feat: prepare release pipelines

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

It adds two new pipelines to prepare major/minor and patch releases.

## Why is it important?

This automates a tough process to modify some files and make PRs with the changes.

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

**Review by mdelapenya — APPROVED** [review]

File: .ci/jobs/beats-release-patch.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+- job:
+    name: Beats/Release/netas-release-patch
```
Comment (mdelapenya) [inline .ci/jobs/beats-release-patch.yml:3]: `netas`?

**Review by v1v — APPROVED** [review]
Once the `netas` comment is answered :)

File: .ci/jobs/beats-release-minor-major.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+- job:
+    name: Beats/Release/netas-release-minor-major
```
Comment (kuisathaverat) [inline .ci/jobs/beats-release-minor-major.yml:3]:
```suggestion
    name: Beats/Release/beats-release-minor-major
```

File: .ci/jobs/beats-release-patch.yml (hunk 1 above)
Comment (kuisathaverat) [inline .ci/jobs/beats-release-patch.yml:3]:
```suggestion
    name: Beats/Release/beats-release-patch
```

Comment (kuisathaverat) [reply .ci/jobs/beats-release-patch.yml]: 🤦

**Pull request merged**